cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -49,9 +49,31 @@
         </div>
     {% endif %}
 
+    {#
+        Determine how many filters there will be across the first row.
+        If it is 3, we set columns to 1-3 (33%). Otherwise we set
+        columns to 1-2 (50%) width of the available area.
+    #}
+    {% set filter_count = 0 %}
+    {% if value.categories.filter_category %}
+        {% set filter_count = filter_count + 1 %}
+    {% endif %}
+    {% if show_topic_filter %}
+        {% set filter_count = filter_count + 1 %}
+    {% endif %}
+    {% if value.authors %}
+        {% set filter_count = filter_count + 1 %}
+    {% endif %}
+    {% if value.statuses and form.statuses %}
+        {% set filter_count = filter_count + 1 %}
+    {% endif %}
+    {% if value.products %}
+        {% set filter_count = filter_count + 1 %}
+    {% endif %}
+
```

Comment:
Is there ever a scenario where all five filters are present (and we'd want `1-3`)?

Reply:
Not in the filters I am aware of. Only have zero to three potential combos.